src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2552,6 +2552,23 @@ public partial class DivideByZeroException : System.ArithmeticException
         [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute]
         static double IFloatingPoint<double>.Truncate(double x) { throw null; }
 
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute]
```

Comment:
@pgovind I see a blank line here so perhaps you hand edited? If you generate mechanically then it avoids diffs later: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md#L16

No need to change here though.